.travis.yml

```diff
@@ -5,6 +5,7 @@ branches:
     - master
     - develop
     - /^holochain-nodejs\d+\.\d+\.\d+/
+    - /^holochain-cmd-v\d+\.\d+\.\d+/
```

Comment:
This line and the above line can be removed before this gets merged since they were just needed for testing releases